tests/x11/libreoffice/libreoffice_mainmenu_components.pm

```diff
@@ -92,7 +93,23 @@ sub run {
 
     # launch components from Activities overview
     $self->open_overview();
-    type_string "base";                                #open base
+    type_string "base";
+    # tag is base-install means libreoffice-base not installed
+    if (assert_screen 'base-install')
+    {
+       send_key 'esc';
+       send_key 'esc';
+       x11_start_program('xterm');
+       script_run("gsettings set org.gnome.desktop.session idle-delay 0", 0); #value=0 means never blank screen
+       become_root;
+       prepare_Oss_repo;
+       script_run("zypper in -y libreoffice-base", 900);
+       script_run("gsettings set org.gnome.desktop.session idle-delay 900", 0); #default value=900
+       disable_Oss_repo;
+       send_key 'alt-f4';
+       $self->open_overview();
+       type_string "base";
+    }
```

Comment:
I set blank screen is in order to make sure install complete. For network and other factors in my localhost, I have to set this.